queries.sql

```diff
@@ -1,10 +1,69 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name LIKE '%mon';
-SELECT name FROM animals WHERE EXTRACT(year FROM date_of_birth) BETWEEN 2016 AND 2019;
-SELECT name FROM animals WHERE neutered AND escape_attempts < 3;
-SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
-SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
-SELECT * FROM animals WHERE neutered;
-SELECT * FROM animals WHERE name != 'Gabumon';
-SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you don't remove the previous queries from your project because these queries are related to your next milestone queries so please don't delete them keep all the queries in your project 👍